### PR TITLE
[DRAFT] UIDS-122 - Add yellow color palette and update alerts colors.

### PR DESCRIPTION
**Pull request by fausgaitan**
- Added new yellow color palette to match [figma definition](https://www.figma.com/file/471iS3QrpPtXZpS1xbsJ20/?node-id=1252%3A812)
- Updated classes in `Flash/AlertMessage.jsx` to resolve styles override issues
- Updated `Flash/AlertMessage.scss ` with new color variables. ([figma](https://www.figma.com/file/471iS3QrpPtXZpS1xbsJ20/?node-id=1276%3A14))
- Updated `src/Popper/Popper.scss` to improve tooltip look & feel.

### CODE REVIEW

File: scss/colors/palette.scss [hunk 1]
```diff
@@ -57,6 +57,16 @@ $ux-orange-700: #AE6608;
 $ux-orange-800: #7D4A06;
 $ux-orange-900: #653B05;
 
+$ux-yellow-900: #655505;
```
Comment (davidmferris) [inline scss/colors/palette.scss:60]: Nitpick, but could we move these below the red colors to preserve the alphabetization as well as make them ascending 100-900? Also looks like there is another $ux-yellow-100 in this file so there may be a conflict.

File: src/Popper/Popper.scss [hunk 2]
```diff
@@ -10,15 +10,15 @@
   letter-spacing: normal;
   padding: 0.75rem;
   z-index: $z-index-flash;
+  font-weight: $font-weight-light;
 
   &__header {
-    border-bottom: 1px solid $ux-gray-300;
-    margin-bottom: 0.5rem;
+    margin-bottom: 0.3rem;
+    font-weight: $font-weight-bold;
```
Comment (davidmferris) [inline src/Popper/Popper.scss:17]: Could this use `@include font-type-30--bold;`?

**Review by davidmferris — CHANGES_REQUESTED** [review]
Looks great! Just a couple of small comments.

File: scss/colors/palette.scss (hunk 1 above)
Comment (fausgaitan) [reply scss/colors/palette.scss]: done! 71bdc1bd91254377f5ebe09384f25b7efffdad2c

File: src/Popper/Popper.scss (hunk 2 above)
Comment (fausgaitan) [reply src/Popper/Popper.scss]: done! 0578ea0fa5a9cfd2d5ffcb2887a4a6644707381d

**Review by davidmferris — APPROVED** [review]
Looks great!

**Pull request merged**